main.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/gocarina/gocsv"
+	"github.com/gorilla/mux"
+)
+
+type Pokemon struct {
+	ID         string `csv:"ID"`
+	Name       string `csv:"Name"`
+	Type_1     string `csv:"Type_1"`
+	Type_2     string `csv:"Type_2"`
+	Total      int    `csv:"Total"`
+	HP         int    `csv:"HP"`
+	Attack     int    `csv:"Attack"`
+	Defense    int    `csv:"Defense"`
+	Sp_Atk     int    `csv:"Sp_Atk"`
+	Sp_Def     int    `csv:"Sp_Def"`
+	Speed      int    `csv:"Speed"`
+	Generation int    `csv:"Generation"`
+	Legendary  bool   `csv:"Legendary"`
+}
+
+func getAll(w http.ResponseWriter, req *http.Request) {
+	content, err := os.Open("data.csv")
```

Comment:
Please avoid the use of `magic naming` (unnamed constants).

To have in mind. What if you decide to change the `data.csv` file?  Please give it a look to [spf13/viper](https://github.com/spf13/viper).